python/ray/dag/tests/experimental/test_execution_schedule_gpu.py

```diff
@@ -372,6 +382,71 @@ def test_three_actors_with_nccl_2(ray_start_regular, single_fetch, monkeypatch):
     compiled_dag.teardown()
 
 
+@pytest.mark.parametrize("ray_start_regular", [{"num_gpus": 3}], indirect=True)
+@pytest.mark.parametrize("overlap_gpu_communication", [True, False])
+def test_overlap_gpu_communication(ray_start_regular, overlap_gpu_communication):
+    if not USE_GPU:
```

Comment:
Please update this to not require GPUs. You can mock them out or just unit-test the DAG operation scheduling part.

Reply:
Other tests in the same file also require GPU. I created a ticket as a follow-up to do this for the entire test file: https://github.com/ray-project/ray/issues/48459